src/test/java/redis/clients/jedis/misc/AutomaticFailoverTest.java

```diff
@@ -34,9 +26,10 @@ public class AutomaticFailoverTest {
 
   private static final Logger log = LoggerFactory.getLogger(AutomaticFailoverTest.class);
 
-  private final HostAndPort hostPort_1 = new HostAndPort(HostAndPorts.getRedisServers().get(0).getHost(), 6378);
-  private final HostAndPort hostPort_1_2 = HostAndPorts.getRedisServers().get(0);
-  private final HostAndPort hostPort_2 = HostAndPorts.getRedisServers().get(7);
+  // TODO(imalinovskyi): Figure out how we deploy this endpoint
+  private final HostAndPort hostPort_0 = new HostAndPort(HostAndPorts.getRedisEndpoint("standalone0").getHost(), 6378);
+  private final EndpointConfig endpointStandalone0 = HostAndPorts.getRedisEndpoint("standalone0");
+  private final EndpointConfig endpointStandalone7 = HostAndPorts.getRedisEndpoint("standalone7-with-lfu-policy");
```

Comment:
My intention behind previous names:
* hostPort_1 - first node / proxy / cluster (which is failing); in internal implementation would have priority=1
* hostPort_1_2 - first node / proxy / cluster (not failing, used in only 1 test); in internal implementation would have priority=1
* hostPort_2 - second node / proxy / cluster (to failover to); in internal implementation would have priority=2

Can we have better names when using EndpointConfig?

Reply:
Improved